### PR TITLE
Remove lint and unittest task from the release pipeline

### LINKED ISSUE #1018: Remove lint task from the release pipeline
Since we already do linting on every PR there is really no advantages having it
at release (and the release is already quite long to do with the binary building
for multiple platforms).

/kind cleanup
/assign

### PR DESCRIPTION
**Pull request by chmouel**
# Changes

We already do it on PR

Closes #1018

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [X] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [X] Run the code checkers with `make check`
- [X] Regenerate the manpages, docs and go formatting with `make generated`
- [X] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/cli/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

**Comment by vdemeester** [comment]
I think we should keep unit tests. I am ok with removing linting but I would really like still having the unit testing part.

**Comment by chmouel** [comment]
I forgot about this, repushing with unit-tests,

File: tekton/release.sh [hunk 1]
```diff
@@ -8,7 +8,7 @@ TARGET_NAMESPACE="release"
 SECRET_NAME=bot-token-github
 PUSH_REMOTE="${PUSH_REMOTE:-${UPSTREAM_REMOTE}}" # export PUSH_REMOTE to your own for testing
 
-CATALOG_TASKS="lint build tests"
+CATALOG_TASKS="build"
```
Comment (piyush-garg) [inline tekton/release.sh:11]: as decided to keep unit tests, we should no remove tests tasks and after renaming it should be `test`

**Comment by chmouel** [comment]
/close

seems like people dont want it :-p

**Pull request closed without merging**